.zprintrc

```diff
@@ -0,0 +1,50 @@
+;; -*- mode: clojure -*-
+;; vi: ft=clojure
+{:width     105
+ :style     [;; community style
+             ;; https://github.com/kkinnear/zprint/blob/main/doc/reference.md#community
+             :community
+
+             ;; no comma in map
+             :no-comma
+
+             :custom-justify
+
+             ;; justify :as, :refer ... in (ns :require (...)) form
+             ;; https://github.com/kkinnear/zprint/blob/main/doc/reference.md#require-justify
+             :require-justify
+
+             ;; respect all newlines
+             ;; https://github.com/kkinnear/zprint/blob/main/doc/reference.md#respect-nl
+             :respect-nl
+
+             ;; respect blank line
+             ;; https://github.com/kkinnear/zprint/blob/main/doc/reference.md#respect-bl
+             :respect-bl]
+ :fn-map
+ {"reg-sub"         :arg1-pair
+  "list-comp"       :binding
+  "defview"         :arg1-body
+  "letsubs"         :binding
```

Comment:
Could you add `"deftest-sub" :arg1-body` as well?

Reply:
done